suite/tests/api/ir_aarch64.c

```diff
@@ -5350,30 +5365,47 @@ test_sys(void *dc)
      */
 
     /* DC ZVA, Xt => SYS #3, C7, C4, #1, Xt */
-    instr = INSTR_CREATE_sys(
-        dc, opnd_create_immed_int(DR_DC_ZVA, OPSZ_2),
-        opnd_create_base_disp_aarch64(DR_REG_X0, DR_REG_NULL, 0, false, 0, 0, OPSZ_sys));
-
-    test_instr_encoding(dc, OP_sys, instr);
+    instr = INSTR_CREATE_dc_zva(dc, opnd_create_reg(DR_REG_X0));
```

Comment:
This is identical to one iteration of the SYS_CACHE loop?  So why have it separate?

Reply:
These are longhand/explicit `INSTR_CREATE_` example calls which are handy for future maintainers and users, rather than being wrapped in a test macro.

Reply:
OK.